apps/explorer/src/hooks/useSearch.ts

```diff
@@ -111,9 +111,10 @@ const getResultsForAddress = async (rpc: JsonRpcProvider, query: string) => {
 export function useSearch(query: string) {
     const rpc = useRpcClient();
 
-    return useQuery(
-        ['search', query],
-        async () => {
+    return useQuery({
+        // eslint-disable-next-line @tanstack/query/exhaustive-deps
```

Comment:
it doesn't look like we need to disable the linter here?

Reply:
Because we pass `rpc` through a function here, it yells at us, and I didn't want to deal with refactoring this or passing `rpc` as a cache key